google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/cloud/flexible/AppEngineFlexibleDeploymentEditorTest.java

```diff
@@ -242,117 +214,6 @@ public void testValidateConfiguration_invalidCloudSdk() {
     }
   }
 
-  public void testValidateConfiguration_blankYaml() {
```

Comment:
They are not - all these tests were removed as part of the removal off the app.yaml / Dockerfile override feature mentioned in the first comment of the PR.

I still have a todo to fix some tests and add more coverage for the changes.